app/app.go

```diff
@@ -912,3 +975,42 @@ func extractRegisters(m sdkmodule.BasicManager, soloRegisters ...encoding.Module
 	}
 	return s
 }
+
+func allStoreKeys() []string {
+	return []string{
+		authtypes.StoreKey, authzkeeper.StoreKey, banktypes.StoreKey, stakingtypes.StoreKey,
+		minttypes.StoreKey, distrtypes.StoreKey, slashingtypes.StoreKey,
+		govtypes.StoreKey, paramstypes.StoreKey, upgradetypes.StoreKey, feegrant.StoreKey,
+		evidencetypes.StoreKey, capabilitytypes.StoreKey,
+		blobstreamtypes.StoreKey,
+		ibctransfertypes.StoreKey,
+		ibchost.StoreKey,
+		packetforwardtypes.StoreKey,
+		icahosttypes.StoreKey,
+		signaltypes.StoreKey,
+		blobtypes.StoreKey,
+	}
+}
+
+// versionedStoreKeys returns the store keys for each app version
+// ... I wish there was an easier way than this (like using the modules which are already versioned)
+func versionedStoreKeys() map[uint64][]string {
```

Comment:
[question] does this mean that to disable blobstream in v2, we have to do:

```go
		{
			Module:      blobstream.NewAppModule(appCodec, app.BlobstreamKeeper),
			FromVersion: v1, ToVersion: v1,
		},
```

and

```diff
		2: {
			authtypes.StoreKey, authzkeeper.StoreKey, banktypes.StoreKey, stakingtypes.StoreKey,
			minttypes.StoreKey, distrtypes.StoreKey, slashingtypes.StoreKey,
			govtypes.StoreKey, upgradetypes.StoreKey, feegrant.StoreKey,
			evidencetypes.StoreKey, capabilitytypes.StoreKey,
-			blobstreamtypes.StoreKey, ibctransfertypes.StoreKey, ibchost.StoreKey,
+			ibctransfertypes.StoreKey, ibchost.StoreKey,
			packetforwardtypes.StoreKey, icahosttypes.StoreKey, signaltypes.StoreKey,
		},
```

Reply:
[nit] it may be easier to read subsequent diffs if all the store keys were on new lines:

```go
		1: {
			authtypes.StoreKey,
			authzkeeper.StoreKey,
			banktypes.StoreKey,
			blobstreamtypes.StoreKey,
			blobtypes.StoreKey,
			capabilitytypes.StoreKey,
			distrtypes.StoreKey,
			evidencetypes.StoreKey,
			feegrant.StoreKey,
			govtypes.StoreKey,
			ibchost.StoreKey,
			ibctransfertypes.StoreKey,
			minttypes.StoreKey,
			slashingtypes.StoreKey,
			stakingtypes.StoreKey,
			upgradetypes.StoreKey,
		},
		2: {
			authtypes.StoreKey,
			authzkeeper.StoreKey,
			banktypes.StoreKey,
			blobstreamtypes.StoreKey,
			capabilitytypes.StoreKey,
			distrtypes.StoreKey,
			evidencetypes.StoreKey,
			feegrant.StoreKey,
			govtypes.StoreKey,
			ibchost.StoreKey,
			ibctransfertypes.StoreKey,
			icahosttypes.StoreKey, // added in v2
			minttypes.StoreKey,
			packetforwardtypes.StoreKey, // added in v2
			signaltypes.StoreKey, // added in v2
			slashingtypes.StoreKey,
			stakingtypes.StoreKey,
			upgradetypes.StoreKey,
		},
```

Reply:
Yup, I think that should be everything. 

I wonder if we can even remove the directory and import "github.com/celestiaorg/celestia-app/x/blobstream"

I can list it out new lines

Reply:
> I wonder if we can even remove the directory and import "github.com/celestiaorg/celestia-app/x/blobstream"

Cool idea! I just checked and it looks like we may have to use the old name still b/c the last published release of celestia-app (v1.8.0) still calls it qgb: https://pkg.go.dev/github.com/celestiaorg/celestia-app@v1.8.0/x/qgb